main.rb

```diff
@@ -31,6 +31,10 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals.push(rental) unless @rentals.include?(rental)
+  end
```

Comment:
As part of the project requirements, kindly make sure the add_rental method should create a new rental (like Rental.new) with the book and date as parameters. The person itself should also be sent as a parameter to create the new rental to help achieve the 'has-many' association.
Kindly check [this link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more information on that. :wink: